tenant/README.md

```diff
@@ -21,12 +21,42 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/multi-tenancy
 
 ## Basic Usage
 
-Use the following cmd to create a tenant CR. A tenant admin namespace `tenant1admin` will be created.
+As a cluster admin, use the following cmd to create a tenant CR. A tenant admin namespace `tenant1admin` will be created.
 ```
 kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/multi-tenancy/master/tenant/config/samples/tenancy_v1alpha1_tenant.yaml
 ```
 
-Use the following cmd to create a tenantnamespace CR. A tenant namespace `t1-ns1` will be created.
+Create a testing service account, say `t1-admin1` in any namespace, to mimic a tenant admin. (This [script]( https://gist.github.com/innovia/fbba8259042f71db98ea8d4ad19bd708) helps to create SA with corresponding KubeConfig file)
+
+Then you can edit the tenant CR and add `t1-admin1` to the tenantAdmins list
```

Comment:
I would suggest we rewrite this demo slightly to create the admin list right at the beginning alongwith creating the tenant CR itself.  Going ahead with this merge for now but we can update the readme flow a bit more afterwards.